packages/apollo-server-core/src/ApolloServer.ts

```diff
@@ -101,7 +101,19 @@ function approximateObjectSize<T>(obj: T): number {
   return Buffer.byteLength(JSON.stringify(obj), 'utf8');
 }
 
-export class ApolloServerBase {
+export interface ApolloServerBaseInt {
```

Comment:
I think a more commonly seen naming pattern here might result in this being `IApolloServerBase`?  Thoughts?